server/routes_create_test.go

```diff
@@ -53,6 +54,8 @@ func (t *responseRecorder) CloseNotify() <-chan bool {
 
 func createRequest(t *testing.T, fn func(*gin.Context), body any) *httptest.ResponseRecorder {
 	t.Helper()
+	// if OLLAMA_MODELS is not set, set it to the temp directory
+	t.Setenv("OLLAMA_MODELS", cmp.Or(os.Getenv("OLLAMA_MODELS"), t.TempDir()))
```

Comment:
any harm in just setting it to `t.TempDir()`?

Reply:
Yes. Some tests set MODELS explicitly so it can check the contents later. Each call to `t.TempDir()` produces a unique directory so setting it to `t.TempDir()` will invalidate any previously set values and break those tests